lib/powerbar.rb

```diff
@@ -308,14 +308,20 @@ def h_done
   end
 
   def terminal_width
+    require 'io/console'
+    rows, cols = IO.console.winsize
+    cols - 1
```

Comment:
Are you sure `cols - 1` is needed on _all_ platforms? 🤔

Reply:
No I am not sure, but #1 says its needed for OS X as well. I can check on a linux machine in a sec.

Reply:
It's not necessary on linux, at least through PuTTY. :/ Not sure what to recommend.

Reply:
I just tested it on OSX (iTerm and Terminal.app), and it's not needed here either.

I would suggest changing the line to cols -= 1 if Gem.win_platform?

Can you test if this works correctly on win and linux? (it does on OSX here)

Reply:
Actually, it doesn't seem to be necessary on OS X either, at least with up-to-date versions of Terminal.app and iTerm2.app